compiler/noirc_evaluator/src/brillig/brillig_gen/variable_liveness.rs

```diff
@@ -103,11 +103,11 @@ pub(crate) struct VariableLiveness {
     live_in: HashMap<BasicBlockId, Variables>,
     /// The variables that stop being alive after each specific instruction.
     last_uses: HashMap<BasicBlockId, LastUses>,
-    /// The list of block params the given block is defining.
+    /// The vector of block params the given block is defining.
```

Comment:
We do store the values in a Vec but "list" is technically correct here. We look to have renamed list -> vector in multiple comments where "list" is used as a general term.